pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -666,6 +681,9 @@ func shouldRecreateLoadBalancer(current, desired *corev1.Service, platform *conf
 	if platform.Type == configv1.AWSPlatformType && !serviceEIPAllocationsEqual(current, desired) {
 		return true, "its eipAllocations changed"
 	}
+	if platform.Type == configv1.OpenStackPlatformType && !serviceLoadBalancerIPsEqual(current, desired) {
```

Comment:
Do `serviceLoadBalancerIPsEqual` and `getLoadBalancerIPFromService` make the logic any clearer?  To me, they only seem to obfuscate the logic.  
```suggestion
	if platform.Type == configv1.OpenStackPlatformType && current.Spec.LoadBalancerIP != desired.Spec.LoadBalancerIP {
```

Reply:
fine, I just did like other platforms which leverage a function. I agree in this case this is not really needed. I'll remove it.